### PR TITLE
rich: add page

### PR DESCRIPTION
**Pull request by nik-sm**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

1.6.0 - https://github.com/Textualize/rich-cli

### CODE REVIEW

File: pages/common/rich.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# rich
+
+> Rich CLI - fancy terminal display.
+> More information: <https://github.com/Textualize/rich-cli>.
+
+- Show file contents with syntax highlighting:
+
+`rich file.py`
```
Comment (blueskyson) [inline pages/common/rich.md:8]:
```suggestion
`rich {{path/to/file.py}}`
```

File: pages/common/rich.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Add line numbers, and indentation guides:
+
+`rich file.py --line-number --guides`
```
Comment (blueskyson) [inline pages/common/rich.md:12]:
```suggestion
`rich {{path/to/file.py}} --line-number --guides`
```

File: pages/common/rich.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Apply a theme:
+
+`rich file.py --theme monokai`
```
Comment (blueskyson) [inline pages/common/rich.md:16]:
```suggestion
`rich {{path/to/file.py}} --theme {{monokai}}`
```

File: pages/common/rich.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Display file in an interactive pager:
+
+`rich file.py --pager`
```
Comment (blueskyson) [inline pages/common/rich.md:20]:
```suggestion
`rich {{path/to/file.py}} --pager`
```

File: pages/common/rich.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- Display contents from a URL:
+
+`rich https://raw.githubusercontent.com/Textualize/rich-cli/main/README.md --markdown --pager`
```
Comment (blueskyson) [inline pages/common/rich.md:24]:
```suggestion
`rich {{https://raw.githubusercontent.com/Textualize/rich-cli/main/README.md}} --markdown --pager`
```

File: pages/common/rich.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- Export a file as HTML:
+
+`rich file.md -o file.html`
```
Comment (blueskyson) [inline pages/common/rich.md:28]:
```suggestion
`rich {{path/to/file.md}} -o {{path/to/file.html}}`
```

File: pages/common/rich.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+- Print text with formatting tags, custom alignment, and fixed line width:
+
+`rich --print "Hello [green on black]Stylized[/green on black] [bold]World[/bold]" --center --width 10`
```
Comment (blueskyson) [inline pages/common/rich.md:32]:
```suggestion
`rich --print {{"Hello [green on black]Stylized[/green on black] [bold]World[/bold]"}} --center --width {{10}}`
```

**Review by blueskyson — COMMENTED** [review]
Thank you for contribution! We use `{{ ... }}` to highlight user-provided token, here are some suggestions.

**Comment by nik-sm** [comment]
Thanks for the feedback! Let me know if any other changes would be useful

File: pages/common/rich.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
+# rich
+
+> Rich CLI - fancy terminal display.
+> More information: <https://github.com/Textualize/rich-cli>.
+
+- Show file contents with syntax highlighting:
```
Comment (EmilySeville7cfg) [inline pages/common/rich.md:6]:
```suggestion
- Show a specific file with syntax highlighting:
```

File: pages/common/rich.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+`rich {{path/to/file.py}}`
+
+- Add line numbers, and indentation guides:
+
+`rich {{path/to/file.py}} --line-number --guides`
+
+- Apply a theme:
```
Comment (EmilySeville7cfg) [inline pages/common/rich.md:14]:
```suggestion
- Apply a specific theme:
```

File: pages/common/rich.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+`rich {{path/to/file.py}} --theme {{monokai}}`
+
+- Display file in an interactive pager:
```
Comment (EmilySeville7cfg) [inline pages/common/rich.md:18]:
```suggestion
- Display a specific file in an interactive pager:
```

File: pages/common/rich.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+`rich {{path/to/file.py}} --pager`
+
+- Display contents from a URL:
```
Comment (EmilySeville7cfg) [inline pages/common/rich.md:22]:
```suggestion
- Display contents from a specific URL:
```

File: pages/common/rich.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+
+`rich {{https://raw.githubusercontent.com/Textualize/rich-cli/main/README.md}} --markdown --pager`
+
+- Export a file as HTML:
```
Comment (EmilySeville7cfg) [inline pages/common/rich.md:26]:
```suggestion
- Export a specific file as HTML:
```

File: pages/common/rich.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+
+`rich {{path/to/file.md}} -o {{path/to/file.html}}`
+
+- Print text with formatting tags, custom alignment, and fixed line width:
```
Comment (EmilySeville7cfg) [inline pages/common/rich.md:30]:
```suggestion
- Display a text with specific formatting tags, alignment, and line width:
```

File: pages/common/rich.md [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+
+`rich --print {{"Hello [green on black]Stylized[/green on black] [bold]World[/bold]"}} --center --width {{10}}`
```
Comment (EmilySeville7cfg) [inline pages/common/rich.md:32]:
```suggestion
`rich --print {{"Hello [green on black]Stylized[/green on black] [bold]World[/bold]"}} --{{left|center|right}} --width {{10}}`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/rich.md (hunk 8 above)
Comment (EmilySeville7cfg) [reply pages/common/rich.md:6]: @navarroaxel, what about to stick to `Display` word instead of `Print` in all pages?

**Review by blueskyson — APPROVED** [review]

**Comment by nik-sm** [comment]
Hi there - thanks for the feedback! I incorporated some of the suggested changes above.
I removed the word `specific` to follow the grammatical style of a few other sample pages (`tldr cat`, `tldr pip`, `tldr vim`) - let me know what you think.

File: pages/common/rich.md [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+
+`rich {{path/to/file.py}} --pager`
+
+- Display contents from a URL:
+
+`rich {{https://raw.githubusercontent.com/Textualize/rich-cli/main/README.md}} --markdown --pager`
+
+- Export a file as HTML:
+
+`rich {{path/to/file.md}} -o {{path/to/file.html}}`
```
Comment (navarroaxel) [inline pages/common/rich.md:28]: We prefer long arguments if available
```suggestion
`rich {{path/to/file.md}} --export-html {{path/to/file.html}}`
```

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Merged, thanks? 👍🏻

**Pull request merged**